tests/utils/pod.go

```diff
@@ -107,7 +109,7 @@ func FindPodByPrefix(clientSet *kubernetes.Clientset, namespace, prefix, labelSe
 			}
 			return false, fmt.Errorf("Unable to find pod starting with prefix %s", prefix)
```

Comment:
This line `return false, fmt.Errorf("Unable to find pod starting with prefix %s", prefix)`, inside the Poll, is saying both _continue to poll_ (done==false), AND _stop the poll_ (err != nil). The err != nil take precedence so the Poll will stop. Is this what you want?

Reply:
You are right its probably not what I want, but I do want to indicate that I didn't find anything, but I should do it outside of the poll.